__tests__/intergration.test.js

```diff
@@ -0,0 +1,70 @@
+const request=require("supertest")
+const seed=require("../db/seeds/seed.js")
+const db=require("../db/connection.js")
+const data=require("../db/data/test-data")
+const app=require("../app.js")
+
+const fs=require("fs/promises")
```

Comment:
While it is ok to use fs here, it isn't actually necessary as we could take the simpler and more efficient option of simply requiring the JSON file in here